WorkflowRxSwift.podspec

```diff
@@ -19,7 +19,9 @@ Pod::Spec.new do |s|
   s.source_files = 'WorkflowRxSwift/Sources/**/*.swift'
 
   s.dependency 'Workflow', "#{s.version}"
-  s.dependency 'RxSwift', '~> 5.1.1'
+  s.dependency 'RxSwift', '= 4.4.0'
+  s.dependency 'RxCocoa', '= 4.4.0'
+  s.dependency 'RxAtomic', '= 4.4.0'
```

Comment:
Can we use `>= 4.4.0`?

Reply:
I don't see any breaking changes - will switch to `>=`